handler.py

```diff
@@ -1,33 +1,68 @@
 import json
-import datetime
 import logging
+import boto3
+import os
 
 from util.s3_helper import send_to_s3
+from util.validator import validateTarget
```

Comment:
Consider renaming "validTarget" to "TargetValidator", it moves from an action to a noun, which I believe is more OOP proper.

Reply:
Scratch that, I think we should just make use of the existing Target object

Reply:
No worries, happy to discard this. I only created this function to reduce repetition in `handler.py` as eventually there will be multiple ingestion points which will consume a hostname or an IP address.